base/src/main/java/org/aya/core/def/PrimDef.java

```diff
@@ -156,6 +160,29 @@ private final class Initializer {
 
         throw new InternalException("Expected strings but found " + first + " and " + second);
       }
+
+      public final @NotNull PrimDef.PrimSeed I =
+        new PrimSeed(ID.I,
+          ((prim, tyckState) -> PrimTerm.Interval.INSTANCE),
```

Comment:
This function will never be called. I recommend using `throw AssertionError` here

Reply:
Yes please do the same for imin and imax